### PR TITLE
Extract exif data

### PR DESCRIPTION
**Pull request by philippmwirth**
# Extract exif data

Closes #428.

Simply use the `lightly_utils` package to extract exif data and send them to the platform.

Example:

grafik

TODO:
- [x] Release a first version of the `lightly_utils` package
- [x] Re-trigger tests for this branch

### CODE REVIEW

**Comment by IgorSusmelj** [comment]
I like the idea and implementation. But I would not introduce the new dependency until the new package is released. So I would mark this PR as "on hold" or so?

**Comment by philippmwirth** [comment]
> 
> 
> I like the idea and implementation. But I would not introduce the new dependency until the new package is released. So I would mark this PR as "on hold" or so?

I would have just made the release today :) But I can also mark it as a draft for now.

**Review by MalteEbner — APPROVED** [review]
Good implementation, makes the codebase much smaller.

Great that you added more tests for the utils, now our coverage gets even higher :)

File: tests/api_workflow/test_api_workflow_upload_dataset.py [hunk 1]
```diff
@@ -35,10 +42,6 @@ def get_quota_reduced():
         with self.assertRaises(ValueError):
             self.api_workflow_client.upload_dataset(input=self.folder_path)
 
-
-    def test_upload_dataset_from_folder(self):
```
[the remark this answers is not in the document]
Comment (philippmwirth) [reply tests/api_workflow/test_api_workflow_upload_dataset.py:39]: I think I deleted this by accident... Let me add it again :)

**Pull request merged**